Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -736,6 +758,7 @@ void ViewTransition::handleTransitionFrame()
     }
 
     updatePseudoElementStyles();
+    updatePseudoElementSizes();
```

Comment:
Is this slightly more expensive than the previous iteration? Due to looping twice through names instead of once.

Reply:
I guess slightly, yes. The reduction in wasted work massively compensates for that though :)